parse_ci_monitor_json.py

```diff
@@ -8,7 +8,7 @@
 import subprocess
 import pprint
 from datetime import datetime
-from typing import List
+from typing import List, Set
 
 
 def argparser():
```

Comment:
There's a pretty good `argparse` library for Python if you're interested: https://docs.python.org/3/library/argparse.html

Reply:
I actually use that same library. Perhaps not clear here if you just look at the changes in this commit.